docs/src/main/sphinx/connector/loki.md

```diff
@@ -174,3 +174,12 @@ FROM
     )
   )
 ;
+```
+
+The columns have the following names and types 
+
+| Column Name | Type                        |
```

Comment:
Please use a list-table and same formatting as in other data type mapping tables

Also sentence case titles so 

Column names

Reply:
Do you have an example of a data type mapping table?